src/app/language.js

```diff
@@ -158,7 +176,8 @@ App.Localization.langcodes = {
     },
     'ca': {
         name: 'Catalan',
-        nativeName: 'Català'
+        nativeName: 'Català',
+        flag: 'catalan'
```

Comment:
why not flag:'ca' or 'es-ca'? it's a little weird to have some flags using a 2 letter langcode, other full name. Not really important, but might worth considering